docs/basics.md

```diff
@@ -157,6 +157,165 @@ other things yourself, if needed.
 
 [Read more about customising `urql` in the "Extending & Experimenting" section.](extending-and-experimenting.md)
 
+## Server-side rendering
+
+Server-side rendering is a common method to reduce the time it takes for
+a user to see a React page's content. Typically this is implemented using
+the [`react-dom/server` package](https://reactjs.org/docs/react-dom-server.html).
+
+`urql` can be set up to fetch data on the server and rehydrate this data
+on the client, so that the user's browser does not need to refetch it and
+can seamlessly rehydrate your React page.
+
+There are two parts in `urql` that enable server-side rendering:
+
+- The `Client` has a `suspense` option, which enables support for React's
+  experimental Suspense API for data fetching, which allows us to prefetch
+  data before calling `renderToString` or `renderToNodeStream`.
```

Comment:
Can we link to some resources when talking about the SSR process? I can imagine a number of people looking to use URQL for SSR might never have used SSR before and terms such as `renderToString` and `renderToNodeStream` may be new to them.

Reply:
It is linked right at the top of this section:

> Typically this is implemented using
the [`react-dom/server` package](https://reactjs.org/docs/react-dom-server.html).

The problem is I literally couldn't find a good source to link to. As it turns out `reactjs.org` doesn't have ~much~ any content on server-side rendering 😱